### ISSUE TITLE
[dataclasses] I3Position pretty print (Trac #1394)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1386]
Make a prettier printer for I3Position.

Schultz asked for a reminder.  Here it is. :)

Migrated from https://code.icecube.wisc.edu/ticket/1394
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:45", 
    "description": "Make a prettier printer for I3Position.\n\nSchultz asked for a reminder.  Here it is. :)", 
    "reporter": "olivas", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067105393059", 
    "component": "combo core", 
    "summary": "[dataclasses] I3Position pretty print", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2015-10-12T10:25:33", 
    "milestone": "", 
    "owner": "david.schultz", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20151012T10:25:45: 
* olivas changed status from "new" to "assigned"
* olivas changed owner from "" to "david.schultz"

**Comment by icecube-trac** [comment]
Trac update at 20151013T12:55:30: 
* david.schultz changed resolution from "" to "fixed"
* david.schultz commented:

> In changeset 138495:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="138495"
> make a pretty repr for I3Position. fixes #1386
> ```


* david.schultz changed status from "assigned" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:45: 
* olivas commented:

> Milestone IceSim 5.0 deleted


* olivas changed milestone from "IceSim 5.0" to ""
